app/code/Magento/Eav/Model/Entity/Attribute/AbstractAttribute.php

```diff
@@ -788,6 +796,45 @@ public function getFlatColumns()
         return $this->_getFlatColumnsDdlDefinition();
     }
 
+    /**
+     * Processing object after load data
+     *
+     * @return $this
+     */
+    public function _afterLoad()
+    {
+        parent::_afterLoad();
+
+        $this->addFrontendLabels();
+
+        return $this;
+    }
+
+    /**
+     * Add Frontend Labels information
+     *
+     * @throws LocalizedException
+     */
+    public function addFrontendLabels()
+    {
+        $labels = [];
+        foreach($this->_getResource()->getStoreLabelsByAttributeId($this->getId()) as $storeId => $label){
+            $labels[] = $this->getFrontendLabelFactory()->create(['data' => ['store_id' => $storeId, 'label' => $label]]);
+        }
+
+        $this->setFrontendLabels($labels);
+    }
+
+    /**
+     * Returns the frontend label factory
+     *
+     * @return FrontendLabelFactory|mixed
+     */
+    public function getFrontendLabelFactory()
```

Comment:
Please, remove this method, just use `frontendLabelFactory` dependency when needed.